.vuepress/config.js

```diff
@@ -105,6 +105,24 @@ module.exports = {
             '/learn/core_concepts/relevancy',
           ],
         },
+        {
+          title: '⚙️ Configuration',
+          collapsable: false,
+          path: '/learn/configuration/instance_options/',
+          children: [
+            '/learn/configuration/instance_options',
+            {
+              title: 'Index settings',
+              collapsable: false,
```

Comment:
The link still isn’t clickable 👀

You may need to change this to `index_settings` if you decide to rename the page to replace "Settings" with "Index settings"
```suggestion
              collapsable: false,
              path: '/learn/configuration/settings/',
```

Reply:
yeah, I was going to add the link, but then realised that none of our subheadings on the sidebar menus are links. "Inner workings," "Data backup," etc are all regular text, so I'm opting for consistency here.

@bidoubiwa is working on a second docs rebrand PR which will also remove the `:hover` state from these headings, so they won't look like links.